.github/workflows/update-k8s-versions.yml

```diff
@@ -0,0 +1,36 @@
+name: "update-kubernetes-versions"
+on:
+  schedule:
+    # every week on Thursday at 20:30 UTC
+    - cron: "45 20 * * *"
+env:
+  GOPROXY: https://proxy.golang.org
+  GO_VERSION: 1.16.4
+jobs:
+  generate-docs:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{env.GO_VERSION}}
+          stable: true
+      - name: Bump Kuberenetes Versions
+        id: bumpk8s
+        run: |
+          make update-kubernetes-version
+          echo "::set-output name=changes::$(git status --porcelain)"
+      - name: Create PR
+        if: ${{ steps.bumpk8s.outputs.changes != '' }}
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.MINIKUBE_BOT_PAT }}
+          commit-message: bump default/newest kubernetes versions
+          committer: minikube-bot <minikube-bot@google.com>
+          author: minikube-bot <minikube-bot@google.com>
+          branch: auto_bump_k8s_versions
+          push-to-fork: minikube-bot/minikube
+          base: master
+          delete-branch: true
+          title: 'bump default/newest kubernetes versions'
+          body: 'This PR was auto-generated by make update-kubernetes-version using [update-k8s-versions.yml](https://github.com/kubernetes/minikube/tree/master/.github/workflows) CI Workflow. Please only merge if all the tests pass.\n\n${{ steps.bumpk8s.outputs.changes }}'
```

Comment:
the \n\n won't work here, the better thing to do is use the `|` strategy that let's you format more freely